spec/test_equipment/user_operation/user_operation_helper_spec.rb

```diff
@@ -58,6 +58,15 @@
     end
   end
 
+  describe '#switch_to_the_window' do
```

Comment:
Could you implement test about raising timeout error?

Reply:
maybe...
i'll try it!

Reply:
Implementation example:
https://github.com/lifull-dev/bucky-core/blob/master/spec/test_equipment/user_operation/user_operation_helper_spec.rb#L137

Reply:
thanks!
I see and know how to check TimeOutException.

But how to raise TimeOutExeption is bothering me.